src/config/configuration.ts

```diff
@@ -0,0 +1,8 @@
+export default () => ({
+  exchange: {
+    baseUri:
+      process.env.EXCHANGE_API_BASE_URI ||
```

Comment:
This is unrelated to this PR, but as a personal preference I'd say 80 chars per line is too short for JS, I'd increase it to 100, 110 or even 120.

Reply:
Yes I agree. I haven't change the defaults so far but we can review the number of columns 👍